fbw-a32nx/src/systems/instruments/src/ND/style.scss

```
@@ -21,3 +21,7 @@
   background: $display-background;
   fill: rgb(4,4,5);
 }
+
+.TCASStby {
```

```suggestion
.TcasStandbyMessage {
```
We prefer title case, with a special exception for 2 chars only.